packages/developer-console-spa/src/components/AddAppForm.tsx

```diff
@@ -42,7 +42,8 @@ export default function AppAppForm () {
           window.OpNotification.success( { subject: 'App Created Succssfully!' } );
           history.push( res.data.app.appId );
         }
-      } ).catch( err => {
+      } )
+      .catch( err => {
```

Comment:
Add type to err

Reply:
Not needed, as the type is inherited from it's parent.

Reply:
<img width="845" alt="Screenshot 2021-07-30 at 6 44 57 PM" src="https://user-images.githubusercontent.com/19623278/127658174-022d9dbb-acd8-4627-9e5d-78e313c6ae00.png">